langgraph/pregel/__init__.py

```diff
@@ -346,6 +346,11 @@ def _transform(
                 # interrupt if any channel written to is in interrupt list
                 if any(chan for chan, _ in pending_writes if chan in self.interrupt):
                     break
+            else:
+                raise RuntimeError(
+                    f"Recursion limit of {config['recursion_limit']} reached"
+                    "without hitting a stop condition."
```

Comment:
Would provide a recommended invocation if they want to increase. `chain.invoke(..., {"recursion_limit": N})`